persistence/src/main/scala/com/mesosphere/usi/storage/zookeeper/ZooKeeperPersistenceStore.scala

```diff
@@ -0,0 +1,245 @@
+package com.mesosphere.usi.storage.zookeeper
```

Comment:
This should probably be in `persistence-zookeeper` since @meln1k is really keen on creating a `persistence-foundationdb`